dbms/src/Columns/ColumnsCommon.cpp

```diff
@@ -48,10 +52,58 @@ inline UInt64 ToBits64(const Int8 * bytes64)
         | (static_cast<UInt64>(_mm_movemask_epi8(
                _mm_cmpeq_epi8(_mm_loadu_si128(reinterpret_cast<const __m128i *>(bytes64 + 48)), zero16)))
            << 48);
-    return ~res;
+#elif defined(__aarch64__) && defined(__ARM_NEON)
+    const uint8x16_t bitmask
```

Comment:
Do we need some include header file for getting this `uint8x16_t` and other neon instruction?

Reply:
done